docs/app/Examples/elements/List/Types/ListDivided.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import { List } from 'stardust'
+
+const ListDivided = () => {
+  return (
+    <List divided relaxed>
+      <List.Item>
+        <List.Icon name='github' size='large' verticalAlign='middle' />
+        <List.Content>
+          <List.Header as='a'>Semantic-Org/Semantic-UI</List.Header>
+          <List.Description as='a'>Updated 10 mins ago</List.Description>
+        </List.Content>
+      </List.Item>
+      <List.Item>
+        <List.Icon name='github' size='large' verticalAlign='middle' />
+        <List.Content>
+          <List.Header as='a'>Semantic-Org/Semantic-UI-Docs</List.Header>
+          <List.Description as='a'>Updated 22 mins ago</List.Description>
+        </List.Content>
+      </List.Item>
+      <List.Item>
+        <List.Icon name='github' size='large' verticalAlign='middle' />
```

Comment:
We have there hateful situation, `Icon` doesn't have prop `verticalAlign` but `.list .icon` [has](https://github.com/Semantic-Org/UI-List/blob/master/list.css#L386). I don't like idea with `List.Icon`, may be something better?